purchases/src/androidTest/kotlin/com/revenuecat/purchases/BasePurchasesIntegrationTest.kt

```diff
@@ -44,7 +44,7 @@ open class BasePurchasesIntegrationTest {
     protected open val initialForceServerErrors: Boolean = false
     protected open val initialForceSigningErrors: Boolean = false
 
-    protected val testTimeout = 5.seconds
+    protected val testTimeout = 10.seconds
```

Comment:
I thought to only increase it for the tests that were actually failing more often, but we have a couple of those in different flavors, so I thought to just increase it across the board. Lmk what you think